client/post-editor/editor-feedback-invitation/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorFeedbackInvitation( { translate, onTrigger } ) {
+	return (
+		<div className="editor-feedback-invitation">
+			<p className="editor-feedback-invitation__invitation">
+				{ translate(
+					'{{highlight}}Need another set of eyes?{{/highlight}} Ask a friend to review your post.',
+					{
+						components: {
+							highlight: <span className="editor-feedback-invitation__highlight" />
+						}
+					}
+				) }
+			</p>
+			<Button className="editor-feedback-invitation__button" onClick={ onTrigger }>
+				{ translate( 'Send to a Friend' ) }
+			</Button>
+		</div>
+	);
+}
```

Comment:
That's fine. Note too however that even the arrow-function syntax we will have a `displayName` on the React component. I'm fine with using the `function` syntax I just prefer not to. Also are you sure if won't get a name in debugging?

```js
const Test = () => {
	throw new Error( 'hi' );
	return <div>Test</div>;
}
```

<img width="421" alt="screen shot 2017-07-12 at 1 03 26 pm" src="https://user-images.githubusercontent.com/5431237/28114763-83969a30-6702-11e7-9826-a82bee99706c.png">